lib/vsc/utils/fancylogger.py

```diff
@@ -431,7 +440,7 @@ def getRootLoggerName():
         return "not available in optimized mode"
 
 
-def logToScreen(enable=True, handler=None, name=None, stdout=False):
+def logToScreen(enable=True, handler=None, name=None, stdout=False, colorize='never'):
```

Comment:
Just that the magic string `'never'` is spread out in multiple places; it's harder to mistype `None` without noticing in a decent editor.